app/src/styles.css

```diff
@@ -70,6 +65,22 @@ button {
   display: inline-block;
   font-size: 1rem;
   width: 100%;
+  color: #30485f;
+  font-family: -apple-system, BlinkMacSystemFont, "Roboto", "Oxygen", "Ubuntu",
```

Comment:
Since this is for `<img id="userpic">`, the font attributes don't seem to do anything. If a user doesn't have a profile picture, it will generate one for them too.

Reply:
With regards to the comment 2 lines up, you could shorten this CSS block to:
```css
#userpic {
  width: 5rem; 
  margin: 0 0 0.2rem 0;
  display: inline-block; 
}
```
`display:inline-block` might also be okay to drop, but I'll leave that to you!